### PR TITLE
Resolve #22: `--unanimous` will exit(0) and output nothing if all hosts agree.

### LINKED ISSUE #22: Add -u|--unanimous flag: easily identify whether all engines agree on output
To help with automatic comparative testing between engines, add a `-q|--quorum` flag. Behavior will be as follows:

* If all engines under test produce identical output (logic already exists to test this in the collate option), print no output and set exit code to `0`.
* If any engine disagrees, show output, and set exit code to `1`
  * Alternatively: set the exit code to `(1 + the index of the engine in the config file)` to more easily identify which engine disagreed. Perhaps `-1` if more than one engine disagrees or there is a greater than 2-way split on the outputs.

/cc @bterlson

### PR DESCRIPTION
**Pull request by dilijev**
Otherwise, exit(1) and show output as usual.

Resolves #22

### CODE REVIEW

File: bin/eshost.js [hunk 1]
```diff
@@ -45,6 +45,9 @@ const yargv = yargs
   .describe('showSource', 'show input source')
   .boolean('showSource')
   .alias('showSource', 'i')
+  .describe('quorum', 'If all engines agree, exit(0) with no output, otherwise print and exit(1); implies --coalesce')
+  .boolean('quorum')
+  .alias('quorum', 'q')
```
Comment (rwaldron) [inline bin/eshost.js]: I like this, but not sure about the naming. Maybe "unanimous"?

Comment (dilijev) [inline bin/eshost.js]: Yes, good point. That's more accurate to the intent anyway. Short form `-u` is also available.

Comment (rwaldron) [inline bin/eshost.js]: > Short form -u is also available.

Maybe it was meant to be!

**Comment by rwaldron** [comment]
Combined here: https://github.com/bterlson/eshost-cli/pull/37

**Pull request closed without merging**